docs/deployment/getting_started_locally.md

```diff
@@ -373,13 +373,25 @@ The one for the virtual garden is accessible at `./example/operator/virtual-gard
 > ```
 > When the shoot cluster is HA (i.e., `.spec.controlPlane.highAvailability.failureTolerance == zone`), then you can access it via `172.18.255.1`.
 
-Please use this command to tear down your environment:
+Similar as in the section _[Developing Gardener](#developing-gardener)_ it's possible to run a [Skaffold development loop](https://skaffold.dev/docs/workflows/dev/) as well using:
+```shell
+make operator-seed-dev
+```
+
+> :info: Please note that in this setup Skaffold is only watching for changes in the following components:
```

Comment:
I'm not sure whether this will be formatted correctly 🤔

Reply:
I don't think so. At least the preview of the file does not show an icon for `:info:` 😄

Reply:
Fixed, it should be `:information_source` -> ℹ️ just as in https://github.com/gardener/gardener/pull/10630 🙃